compute/oslogin/physical_security_key_ssh.py

```diff
@@ -55,15 +55,23 @@ def main(user_key, ip_address, dryrun, directory=None):
         .getLoginProfile(name="users/{}".format(user_key), view="SECURITY_KEY")
         .execute()
     )
-    security_keys = profile.get("securityKeys")
 
     if "posixAccounts" not in profile:
         print("You don't have a POSIX account configured.")
+        print("Did you enable OSLogin on your compute instance?")
```

Comment:
In this case, are we asking users to enable OS Login for VMs? The answer for the above question could be Yes or No. Can we say something like "Please make sure that you have enabled OS Login for your VM."